CMakeLists.txt

```diff
@@ -284,7 +284,7 @@ endif()
 if(WIN32
 OR EMSCRIPTEN
 OR CMAKE_C_COMPILER_ID STREQUAL "TinyCC"
-OR CMAKE_C_COMPILER_ID STREQUAL "GNU" AND CMAKE_C_COMPILER_VERSION VERSION_LESS 5)
+OR (CMAKE_C_COMPILER_ID STREQUAL "GNU" AND CMAKE_C_COMPILER_VERSION VERSION_LESS 5))
```

Comment:
Ah I think this was the problem with that target being built when it shouldn't! @bnoordhuis

Reply:
"CMake is (mis)designed to evaluate AND and OR at the same precedence, and from left to right."

Reply:
I think the only reasonable response here is :facepalm: